test/user/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+cd $(dirname $0)
+. ../util.sh
+current_test=$(basename $(pwd))
+
+docker compose up -d
+
+user_name=testuser
+docker exec user-alpine-1 adduser --disabled-password "$user_name"
```

Comment:
seems like `docker compose exec` works slightly different, I didn't manage create a user without password, that would be accessible via exec API. I was getting `unable to find user testuser: no matching entries in passwd file` error. so I ended up creating user with `docker exec`